app.js

```diff
@@ -28,6 +31,10 @@ app.patch('/flow/next-step/:uuid', async function(req, res, next) {
           const nextStep = await getNextStep(currentStep);
           await updateActiveStep(subsidyMeasureConsumption, nextStep);
           await generateNotification(subsidyMeasureConsumption, currentStep, nextStep);
+          await updateConsumptionStatus(subsidyMeasureConsumption, CONSUMPTION_STATUS.ACTIVE);
+          if(await isLastStep(currentStep)) {
+            await updateConsumptionStatus(subsidyMeasureConsumption, CONSUMPTION_STATUS.SENT);
+          }
```

Comment:
Couldn't we do as below, to avoid this intermediate status modification to active when not required ? 
```
if(await isLastStep(currentStep)) {
  await updateConsumptionStatus(subsidyMeasureConsumption, CONSUMPTION_STATUS.SENT);
} else {
  await updateConsumptionStatus(subsidyMeasureConsumption, CONSUMPTION_STATUS.ACTIVE);
}
```